src/ng-select/ng-select.component.ts

```diff
@@ -592,6 +593,9 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
 
     private _handleTab(_: KeyboardEvent) {
         if (this.isOpen) {
+            if (this.selectOnBlur && this.itemsList.markedItem) {
+                this.select(this.itemsList.markedItem);
+            }
```

Comment:
@granteagon See discussion here https://github.com/ng-select/ng-select/pull/391. We agreed to call input [selectOnTab] 👍